src/controllers/audience/audienceController.ts

```diff
@@ -4,13 +4,15 @@ import {LearningCatalogue} from '../../learning-catalogue'
 import {Audience} from '../../learning-catalogue/model/audience'
```

Comment:
Again, not in the changes, but setOrganisation redirects to configure-audience without and audienceId, so the links won't work on that page